docs/DeveloperGuide.md

```diff
@@ -264,6 +265,30 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+**Use case: Edit a Client**
+
+**MSS**
+
+1.  User requests to list Clients
+2.  FitEgo shows a list of Clients
```

Comment:
Is this more of a "List a Client" use case?

Reply:
Is it better if I change it into:
    Use case: UC03 - Edit a Client
    MSS:
        1. User requests a list of Clients using `clist` (UC02) or `cfind` (UC04) 
        ..the rest